classes/Adapters/MySQL/ActiveRecordAdapter.php

```diff
@@ -595,4 +615,23 @@ public function GetReportGrouped($_siteId, $_startTimestamp, $_endTimestamp, $_a
 
         return $grouped_types;
     }
+
+    /**
+     * Create temp table `tmp_users` in the external DB
+     * It used in the query of the above function
+     */
+    private function TempUsers()
+    {
+        $_wpdb = $this->connection;
+        $sql = "CREATE TEMPORARY TABLE IF NOT EXISTS tmp_users (ID BIGINT NOT NULL AUTO_INCREMENT, user_login VARCHAR(60) NOT NULL, PRIMARY KEY (ID))";
```

Comment:
@carmelopreeostudios let's avoid using something like tmp_users - I believe all tables have the wordpress prefix & it's better we have wsal_tmp_users to make sure this is our table and no one else will have created it already